packages/common/src/templates/configuration/sandbox/ui.tsx

```diff
@@ -1,7 +1,7 @@
-import * as React from 'react';
+import React from 'react';
 import sortBy from 'lodash/sortBy';
 
-import * as templates from '../../../templates';
+import templates from '../../../templates';
```

Comment:
```suggestion
import * as templates from '../../../templates';
```

I think this one should stay the same, since we want all exports.

Reply:
I think this is also the cause of the build error.